.env.sample

```diff
@@ -97,6 +97,7 @@ EXTERNAL_API_KEY=TESTAPIKEY
 AUDIT_API_URL=
 AUDIT_API_USERNAME=
 AUDIT_API_PASSWORD=
+DELETION_AUDIT_LOGS_DELETE_AFTER_SECONDS=604800
```

Comment:
This names a bit of a mouthful happy to workshop
- `AUDIT_LOGS_EXPIRY_SECONDS` doesnt really get across that its for the `deletion-audit-logs` collection
- `DELETION_AUDIT_LOGS_EXPIRY_SECONDS`  is may better but Its still a mouthful

Reply:
Think it should be clear it's in seconds (ms would be very plausible)